src/Aquifer.Public.API/Endpoints/Resources/Collections/List/Request.cs

```diff
@@ -0,0 +1,27 @@
+using Aquifer.Data.Entities;
+using System.ComponentModel;
+
+namespace Aquifer.Public.API.Endpoints.Resources.Collections.List;
+
+public sealed class Request
+{
+    public const int MaximumLimit = 100;
+
+    /// <summary>
+    /// The optional type of resource to search for, such as "Dictionary". If none specified, will default to None.
+    /// </summary>
+    [DefaultValue(ResourceType.None)]
```

Comment:
I copied this from another route but interestingly this shows the default value as `0` in the documentation.  This can be corrected by using the following instead:
```
[DefaultValue(nameof(ResourceType.None))]
```
Has anyone else encountered this?  I'm wondering if there's a tooling solution to automatically `ToString()` any enum in default value documentation.

Reply:
Maybe could do something in `DefaultParameterOperationProcessor.cs`